scripts/releaseChangelog.js

```diff
@@ -143,7 +143,7 @@ async function main(argv) {
   const changelog = `
 ## TODO RELEASE NAME
 <!-- generated comparing ${lastRelease}..${release} -->
-_${nowFormated}_
+###### _${nowFormated}_
```

Comment:
This breaks the heading order. We would not accept it in the docs or any component.

Reply:
True, my reasoning was that the changelog is rendering into HTML in GitHub only, where they don't respect the heading order in the first place (e.g. multiple h1). I'm updating it.